examples/meta/generator/targets/java.json

```diff
@@ -38,6 +38,18 @@
         "RealMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)",
         "LongRealMatrix": "DoubleMatrix $name = new DoubleMatrix($arguments)"
     },
+    "ListInit": {
+        "BoolVector": "BoolVector $name = new BoolVector({$arguments})",
```

Comment:
not sure how to do this one

Reply:
Don’t know either, last resort would be to translate it to explicit assignment although that would suck